kinetic/distribution.yaml

```diff
@@ -6722,6 +6722,16 @@ repositories:
       url: https://github.com/xuefengchang/micros_swarm_framework.git
       version: master
     status: developed
+  microstrain_3dm_gx5_45:
```

Comment:
It's not referenced elsewhere, but it would be good to change the name at this level too for easier searching and consistency.

Also in the other 2 distros.

Reply:
Thank you for the catch.  Changed the reference to "microstrain_mips" so that naming is consistent.